examples/text_labels_and_annotations/font_family_rc.py

```diff
@@ -23,15 +23,33 @@
 
   rcParams['font.sans-serif'] = ['Tahoma', 'DejaVu Sans',
                                  'Lucida Grande', 'Verdana']
+
+.. redirect-from:: /examples/font_family_rc_sgskip
+
+First example:
 """
 
 import matplotlib.pyplot as plt
 
-plt.rcParams['font.family'] = 'sans-serif'
-plt.rcParams['font.sans-serif'] = ['Tahoma']
+plt.rcParams["font.family"] = "sans-serif"
+plt.rcParams["font.sans-serif"] = ["Tahoma"]
+
+fig, ax = plt.subplots(figsize=(6, 1), facecolor="#eefade")
+ax.text(0.01, 0.2, "Hello World! 01", size=40)
+ax.axis("off")
+plt.show()
+
+
+#############################################################################
+#
+# Second example:
 
-fig, ax = plt.subplots()
-ax.plot([1, 2, 3], label='test')
 
-ax.legend()
+plt.rcParams["font.family"] = "monospace"
+# plt.rcParams['font.monospace'] = ['Computer Modern Typewriter'] # this line gives an error currently, therefore the
+# question: How can one access the fonts from https://github.com/matplotlib/matplotlib/blob/main/lib/matplotlib/mpl-data/matplotlibrc#L273
```

Comment:
This is only a list of font names to try (similar to CSS). There is no guarantee that they are available on your machine, but the list is chosen so that it's highly likely that your machine will have at least one of them installed. It's usually not a good idea to set this to a single element list (unless you don't need portability). I suggest to make this the first example, not set `plt.rcParams['font.monospace']` and explain that it will choose the first available font from `plt.rcParams['font.monospace']`.
Then you can discuss that it's possible to explicitly override that list in the second example (where we are apparently lucky that the doc build system contains the Tahoma font.

Reply:
Interesting, thanks for demystifying this for me!
I think it would be great to have three examples then:
1. plt.rcParams["font.family"] = "sans-serif"
<img width="285" alt="image" src="https://user-images.githubusercontent.com/44469195/165158141-516c497d-bb03-437f-a442-2da0fe717320.png">

2. plt.rcParams["font.monospace"] = "monospace"
<img width="317" alt="image" src="https://user-images.githubusercontent.com/44469195/165158092-f3c7bbf9-b54e-4910-a94f-6f064e75fcea.png">
3. plt.rcParams["font.monospace"] = "monospace" & plt.rcParams["font.monospace"] = "Menlo"
<img width="318" alt="image" src="https://user-images.githubusercontent.com/44469195/165158059-c27ff7f8-b7f0-46f0-a67f-f1b0e8e33eb9.png">